### PR TITLE
Add "shared-directory-xdist" post

### LINKED ISSUE #17: Sharing data between master and workers in xdist
Hi @hackebrot!

First of all congratulations on the idea of creating a pytest-tricks blog! 

I recently [posted](https://github.com/pytest-dev/pytest/issues/1402#issuecomment-186299177) a tip on how to share data between master and worker processes in xdist, I think it might be worth a blog post as I find that a neat trick. :smile:

### PR DESCRIPTION
**Pull request by nicoddemus**
Close #17

### CODE REVIEW

**Comment by hackebrot** [comment]
CI breaks because we don't want to auto-publish. That's kinda confusing actually.

Thank you for submitting this PR! I think there's an import missing? ``tempdir``

File: content/shared_directory_xdist/contents.lr [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+xdist hook can be accessed by the worker nodes later. You can put any
+simple builtin type, like lists, strings, tuples, etc.
+
+The `shared_directory` fixture can then be used by a plugin or even tests
+to have a common directory where information can be exchanged. This
+recipe also shows how one can find out if the code is running in the master
+or a worker node.
+
+(this recipe appeared originally in
+[this pytest issue](https://github.com/pytest-dev/pytest/issues/1402#issuecomment-186299177)).
```
Comment (hackebrot) [inline content/shared_directory_xdist/contents.lr]: I think the dot ``.`` at the end is redundant LOL :grin:

Comment (nicoddemus) [inline content/shared_directory_xdist/contents.lr]: Agreed!

**Comment by nicoddemus** [comment]
Fixed import and removed the dot from the end of the sentence as suggested. :grin:

File: content/shared_directory_xdist/contents.lr [hunk 2]
```diff
@@ -0,0 +1,78 @@
+name: Shared directory between master and workers with pytest-xdist
```
Comment (hackebrot) [inline content/shared_directory_xdist/contents.lr]: Sorry, one last thing. IMHO titlecased posts look really nice.

**Shared Directory Between Master and Workers With pytest-xdist**

```no-highlighting
pip install titlecase
python -c "from titlecase import titlecase; print(titlecase('hello world'))"
```

Comment (hackebrot) [inline content/shared_directory_xdist/contents.lr]: http://www.pydanny.com/titlecasing-markdown-headers-with-python.html :relaxed:

Comment (nicoddemus) [inline content/shared_directory_xdist/contents.lr]: Sure, didn't notice it! :grin:

File: content/shared_directory_xdist/contents.lr [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+def pytest_configure(config):
+    if is_master(config):
+        config.shared_directory = tempdir.mktemp()
+
+
+def pytest_unconfigure(config):
+    if is_master(config):
+        shutil.rmtree(config.shared_directory)
```
Comment (hackebrot) [inline content/shared_directory_xdist/contents.lr]: shutil import :speak_no_evil:

**Pull request merged**